src/components/SmartExcelButton.tsx

```
@@ -0,0 +1,335 @@
+  CircularProgress,
+  Fade,
+  Tooltip,
+  Snackbar,
+  Alert
+} from '@mui/material';
+import {
+  FileDownload,
+  FileUpload,
+  TableView,
+  CheckCircle,
+  Error,
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)